evm/src/executor/inspector/cheatcodes/env.rs

```diff
@@ -156,6 +156,21 @@ pub fn apply<DB: Database>(
             Ok(Bytes::new())
         }
         HEVMCalls::Accesses(inner) => Ok(accesses(state, inner.0)),
+        HEVMCalls::SetNonce(inner) => {
+            data.subroutine
+                .state()
+                .entry(inner.0)
+                .and_modify(|acc| acc.info.nonce = inner.1)
+                .or_insert_with(|| {
+                    let account_info = revm::AccountInfo { nonce: inner.1, ..Default::default() };
+                    Account::from(account_info)
+                });
+            Ok(Bytes::new())
+        }
+        HEVMCalls::GetNonce(inner) => match data.subroutine.state().get(&inner.0) {
+            Some(account) => Ok(abi::encode(&[Token::Uint(U256::from(account.info.nonce))]).into()),
```

Comment:
```solidity
   // Set nonce for an account
    function setNonce(address,uint64);
    // Get nonce for an account
    function getNonce(address) external returns(uint64);
```

Not sure `U256::from` is necessary here, since return is u64? Otherwise lgtm